bors.toml

```diff
@@ -0,0 +1,11 @@
+status = [
+	"continuous-integration/travis-ci/push",
+	"continuous-integration/appveyor/branch"
+]
+pr_status = [
+	"continuous-integration/travis-ci/pr",
+	"continuous-integration/appveyor/pr"
+]
+timeout_sec = 7200
```

Comment:
The maximum time limit in travis is 50 minutes so 7200 is kind of overkill, I guess

Reply:
That is for appveyor.